src/components/RedeemSuccess.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import PropTypes from 'prop-types';
+
+import withIntl from '../lib/withIntl';
+
+import { Flex, Box } from 'grid-styled';
+import { FormattedMessage } from 'react-intl';
+import { P, H5 } from './Text';
+import styled from 'styled-components';
+
+const Title = styled(H5)`
```

Comment:
If you just want to alias the tag, `const Title = H5` should be fine, rather than creating a whole new object.